lib/endpoints/class-wp-rest-controller.php

```diff
@@ -127,10 +127,12 @@ public function delete_item( $request ) {
 	/**
 	 * Prepare the item for create or update operation.
 	 *
-	 * @param WP_REST_Request $request Request object.
-	 * @return WP_Error|object $prepared_item
+	 * @param WP_REST_Request $request  Request object.
+	 * @param bool            $creating Whether the item is being created.
```

Comment:
`wp_insert_post` uses `$updating` instead of `$creating`:

```
/**
 * Fires once a post has been saved.
 *
 * @since 2.0.0
 *
 * @param int     $post_ID Post ID.
 * @param WP_Post $post    Post object.
 * @param bool    $update  Whether this is an existing post being updated or not.
 */
do_action( 'wp_insert_post', $post_ID, $post, $update );
```

Can we follow this pre-existing pattern?

Also, do we need to necessarily pass a variable to the method, or is there a way of determining whether it's a create or update already from within the method? I think it might be safe to key off whether or not the ID is present on the prepared database object.